### PR TITLE
Add test cases for default None

### PR DESCRIPTION
**Pull request by Kludex**

### CODE REVIEW

File: tests/integration/cases/add_none.py [hunk 1]
```diff
@@ -54,6 +56,8 @@
                 "    m: Optional[int] = Field(None, lt=10)",
                 "    n: Optional[int] = Field(default=...)",
                 "    o: Optional[int] = Field(default=None)",
+                "    p: Optional[int] = Field(None)",
+                # "    q: Optional[int] = Field(None, lt=10)",
```
Comment (Kludex) [inline tests/integration/cases/add_none.py:60]:
```suggestion
                "    q: Optional[int] = Field(None, lt=10)",
```
